plugins/sfEacPlugin/lib/sfEacPlugin.class.php

```diff
@@ -1049,4 +1058,36 @@ public static function renderDates($item)
 
     return $dates;
   }
+
+  /**
+   * Determine if one or more description elements are filled in.
+   *
+   * It was either this or a massive conditional statement like:
+   * (isset($a) && $a) || (isset($b) && $b) || ...).
+   *
+   * @param $resource  The actor we're getting information from
+   * @return  bool  True if there are one or more description elements, false otherwise
+   */
+  public function hasDescriptionElements($resource)
+  {
+    // Use @ to coerce non-isset properties to NULL
+    $descriptionElements = @array(
+      $this->existDates,
+      $resource->places,
+      $resource->legalStatus,
+      $resource->functions,
+      $resource->mandates,
+      $this->structureOrGenealogy,
+      $this->generalContext,
+      $this->biogHist
+    );
+
+    $result = false;
```

Comment:
I think you could replace lines 1085-1091 with this use of a weird PHP function (that removes empty elements):

`return (bool)array_filter($a);`